next/modules/sections/PartnersSection.tsx

```diff
@@ -16,9 +16,9 @@ const PartnersSection = () => {
   return (
     <>
       {data?.featuredPartners && data.featuredPartners.data.length > 0 && (
-        <div className="mt-6 grid grid-cols-1 gap-5 md:grid-cols-3">
+        <ul className="mt-6 grid grid-cols-1 gap-5 md:grid-cols-3">
           {data.featuredPartners.data.map((partner) => (
-            <Partner
+            <PartnerCard
               key={partner?.attributes?.title}
```

Comment:
`key`s should alway be unique, so `partner.id` is what we want to use. If there is no unique field, we can use, we usex `index` (with disabled eslint rule) witch is better than potentially duplicate keys.